server/src/main.rs

```diff
@@ -1,14 +1,131 @@
-//! gRPC server implementation
+//! <center>
+//! <img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+//! </center>
+//! <div align="center">
+//!     <a href="https://github.com/Arrow-air/svc-template-rust/releases">
+//!         <img src="https://img.shields.io/github/v/release/Arrow-air/svc-template-rust?sort=semver&color=green" alt="GitHub stable release (latest by date)">
+//!     </a>
+//!     <a href="https://github.com/Arrow-air/svc-template-rust/releases">
+//!         <img src="https://img.shields.io/github/v/release/Arrow-air/svc-template-rust?include_prereleases" alt="GitHub release (latest by date including pre-releases)">
+//!     </a>
+//!     <a href="https://github.com/Arrow-air/svc-template-rust/tree/main">
+//!         <img src="https://github.com/arrow-air/svc-template-rust/actions/workflows/rust_ci.yml/badge.svg?branch=main" alt="Rust Checks">
+//!     </a>
+//!     <a href="https://discord.com/invite/arrow">
+//!         <img src="https://img.shields.io/discord/853833144037277726?style=plastic" alt="Arrow DAO Discord">
+//!     </a>
+//!     <br><br>
+//! </div>
+//!
+//! `svc-template-rust` is responsible for FIXME
 
```

Comment:
I prefer using an include for the docs like I've done for svc-storage:
https://github.com/Arrow-air/svc-storage/blob/develop/server/src/lib.rs